utils/src/main/kotlin/Extensions.kt

```diff
@@ -368,6 +368,11 @@ fun String.percentEncode(): String =
 fun String.stripCredentialsFromUrl() =
     toUri { URI(it.scheme, null, it.host, it.port, it.path, it.query, it.fragment).toString() }.getOrDefault(this)
 
+/**
+ * Return this string lower-cased except for the first character which is upper-cased.
+ */
+fun String.titlecase() = lowercase().uppercaseFirstChar()
```

Comment:
It might be (slightly) confusing that there are different implementations of `titlecase()`. I assume, a central one is not possible because of restrictions regarding the dependencies between modules. But shouldn't then all implementations look the same?

Reply:
> I assume, a central one is not possible because of restrictions regarding the dependencies between modules.

Exactly.

> But shouldn't then all implementations look the same?

I introduced the `uppercaseFirstChar()` utility function only later as it was also needed separately, but only in the main code. I'm undecided et whether it's worth it to duplicate this function also in the independent modules if it's only ever used once there (and thus could be inlined).